chapter03/Task01.py

```diff
@@ -29,10 +29,12 @@
     guess = int(input("Take a guess: "))
     tries += 1
     if tries == 10:
-        print('Ты не справился со мной, машины победили')
+        print('You lose! Machines WON!!! HA-HA-HA! ')
```

Comment:
If the right guess happened on the 10th trial, the problem still thinks that the player lost.

Reply:
Also if the right guess happened during the first trial, "congrats" message won't be printed.

To avoid these mistakes consider moving reading user input to a single place and not have it in different places (before the loop and within).

Reply:
I decided to add a line of output in case of a miracle to the beginning. Without breaking the existing program structure